source/installation-guide.rst

```diff
@@ -147,186 +162,52 @@ instance: ::
 
 .. note::
 
-  All docker, docker-compose commands may need sudo to work.
+  All Docker, Docker Compose commands may need sudo to work.
 
-  To allow non-root users to manage docker, please check docker's documentation:
+  To allow non-root users to manage Docker, please check Docker's documentation:
 
   https://docs.docker.com/engine/installation/linux/linux-postinstall/
 
-Usage
-^^^^^
-
-The web interface is available at ``http://localhost:8000``. The user is
-``admin`` and the password is ``admin``. You also can interact with platform
-using the :doc:`./components-and-apis`.
-
-Read the :doc:`using-api-interface` and :doc:`using-web-interface` for more
-information about how to interact with the platform.
-
-Kubernetes
-----------
-
-.. raw:: html
-
-    <iframe id="ytplayer" type="text/html" width="720" height="405"
-    src="https://www.youtube.com/embed/qGUlBCmmIao?rel=0" frameborder="0"
-    allowfullscreen></iframe>
-
-In this video tutorial above, version v0.4.2 is used, but the same video is valid for the current version, it is only necessary to change to version v0.4.3
-
-
-This section provides instructions on how to create a dojot deployment
-on a multi-node environment, using Kubernetes as the orchestration
-platform.
-
-This deployment option when properly configured can be used for
-creating production environments.
-
-The following sections describe all dependencies and steps required
-for this deployment.
-
-
-Kubernetes Cluster
-^^^^^^^^^^^^^^^^^^
-
-For this guide it is advised that you already have a working K8s cluster.
-
-If you need to build a Kubernetes cluster from scratch, up to date
-information and installation procedures can be found at `Kubernetes setup
-documentation`_.
-
-.. _Kubernetes setup documentation: https://kubernetes.io/docs/setup/
-
-Kubernetes Requirements
-^^^^^^^^^^^^^^^^^^^^^^^
+Volumes
+^^^^^^^
 
-- The Kubernetes supported version is between **v1.11** and **v1.15** .
-- Access to Docker Hub repositories
-- (optional) a storage class that will be used for persistent storage
+When we deploy dojot with the command 'docker-compose up -d' the volumes are enabled and created
+by default.
 
-dojot Deployment
-^^^^^^^^^^^^^^^^
+The volumes of microservices that Dojot uses can be incompatible between dojot versions. This means
```

Comment:
Something confusing, but which has been adopted is to write the word _dojot_ entirely in lower case unless it is at the beginning of the sentence. I did not understand the real reason for this, but in this sentence it is possible to find both forms of writing (Dojot and dojot).